project-icp/bee-pollinator-app
# Cleanup celery

# Overview

We've experienced inability to periodically connect with Celery workers, which were previously queried for their existence.  The problem was difficult to debug and transient, but to preclude problems related to the complex routing and queuing we were engaged in (inherited from old MMW), I've removed that special handling.  It's unclear if this will actually resolve the errors.

I also updated the JRE and Nginx role version as general purpose updates.

Connects #276 

## Demo
#### Improved task submission response time
Before
![screenshot from 2017-09-11 16 47 28](https://user-images.githubusercontent.com/1014341/30296352-dd56c01a-9711-11e7-8ba4-acd0ec3d14c0.png)

After
![screenshot from 2017-09-11 16 47 03](https://user-images.githubusercontent.com/1014341/30296346-d6d6249c-9711-11e7-8653-8411637ae454.png)


#### gzip headers
```
HTTP/1.1 200 OK
Server: nginx
Date: Mon, 11 Sep 2017 20:47:47 GMT
Content-Type: application/javascript
Last-Modified: Mon, 11 Sep 2017 20:39:34 GMT
Transfer-Encoding: chunked
Connection: keep-alive
Vary: Accept-Encoding
ETag: W/"59b6f486-935589"
Content-Encoding: gzip
```
## Testing Instructions
1. Provision app & worker
1. Verify model works and `/yield` endpoint returns in sub 500ms
2. Verify JS content is gzipped
3. Verify that you can provision
